tests/requirements.txt

```
@@ -4,28 +4,36 @@ boto3>=1.17.74
 bottle>=0.12.25
 celery>=5.2.7
 coverage>=5.5
-Django>=4.2.4
-fastapi>=0.92.0
+Django>=4.2.4; python_version < "3.10"
+Django>=5.0; python_version >= "3.10" and python_version < "3.12"
+Django>=5.0a1; python_version >= "3.12"  --pre
+# Dependency orjson has no 3.13 support yet:
+# https://github.com/matyasrichter/fastapi-injector/pull/31
+fastapi>=0.92.0; python_version < "3.13"
 flask>=2.3.2
-grpcio>=1.37.1
-google-cloud-pubsub>=2.0.0
-google-cloud-storage>=1.24.0
+# grpc is not supported on 3.13 yet:
+# https://github.com/grpc/grpc/issues/34922
+grpcio>=1.37.1; python_version < "3.13"
+# depends grpcio on 3.13
+google-cloud-pubsub>=2.0.0; python_version < "3.12"
```

Shouldn't it be 3.13?
```suggestion
google-cloud-pubsub>=2.0.0; python_version < "3.13"
```